en/config/actuators.md

```diff
@@ -523,15 +532,29 @@ If you're using PWM servos, PWM50 is far more common.
 If a high rate servo is _really_ needed, DShot offers better value.
 :::
 
-For each of the control surfaces:
+#### Control surfaces that move both directions about a neutral point
+
+Control surfaces that move either direction around a neutral point include: ailerons, elevons, V-tails, A-tails, and rudders.
+
+To set these up:
 
 1. Set the `Disarmed` value so that the surfaces will stay at neutral position when disarmed.
-   This is usually around `1500` for PWM servos.
+   This is usually around `1500` for PWM servos (near the centre of the servo range)
 2. Move the slider for the surface upwards (positive command) and verify that it moves in the direction defined in the [Control Surface Convention](#control-surface-deflection-convention).
-   - If the control surface moves in the opposite direction, click on the `Rev Range` checkbox to reverse the range.
-3. Move the slider again to the middle and check if the Control Surfaces are aligned in the neutral position of the wing
+
+   - Ailerons, elevons, V-tails, A-tails, and other horizontal surfaces should move up.
+   - Rudders and other "purely vertical" surfaces should move right.
+
+   ::: tip
+   It is important that the slider movement matches the control surface convention, in order to normalize control for different servo mountings (moving the slider up may actually decrease the output value sent to the servo).
+   :::
+
+   If the control surface moves in the opposite direction, click on the `Rev Range` checkbox to reverse the range.
+
+3. Move the slider again to the middle and check if the Control Surfaces are aligned in the neutral position of the wing.
```

Comment:
@sfuhrer Note, we only talk about the fact that they go to the middle. Do we have to worry about the servo having too great a range (i.e. to we need to configure max/min at all?

Reply:
>Do we have to worry about the servo having too great a range (i.e. to we need to configure max/min at all?

Yes that is generally something you have to worry about for each servo configuration.